packages/fxa-auth-server/test/test_server.js

```diff
@@ -26,11 +26,6 @@ function TestServer(config, printLogs, options = {}) {
       determineClientVisibleSubscriptionCapabilities: sinon.fake.resolves(''),
     });
   }
-  if (!Container.has(AuthFirestore)) {
-    Container.set(AuthFirestore, {
-      collection: sinon.fake.returns({}),
-    });
-  }
```

Comment:
I guess you removed this, since you added the check in StripeHelper, which is the only module that depends on it?

Reply:
Hmm maybe I don't understand where this TestServer is used -- when I Shift + F12, I only see it used in scripts/test-remote-quick.js -- I guess some of the remote tests use the StripeHelper, which now starts up the PaymentConfigManager (which now calls `load` in its constructor)?

Reply:
Sounds like Shift+F12 is failing pretty hard there (but these are js not ts sources so maybe that's why).

Also once it's in the container, then that's the ConfigManager.  So even if it's used once (it isn't), if it's the first, then it'd affect the rest of the tests in node process.